From e1c85b6d9c18fee4e65ae6547a138c4165257f84 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Mon, 16 Jun 2025 13:28:54 +0300
Subject: [PATCH] [SW-0] common: fix efuse device macros

There are two EFUSE device instance macros that were defined
with semi-colon. Using EFUSE_DEV_INST in efuse_read_word function
caused a compliationi error. Removing it fixed it.
As a consequence, fix it for EFUSE_SLV_SPI_DEV_INST.

Change-Id: Ib733193560be0a48efdb7d6a0e15f802702f7465
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/include/zephyr/drivers/misc/common/hl_devs.h b/include/zephyr/drivers/misc/common/hl_devs.h
index 85ee42b..e421ab7 100644
--- a/include/zephyr/drivers/misc/common/hl_devs.h
+++ b/include/zephyr/drivers/misc/common/hl_devs.h
@@ -45,10 +45,10 @@
 #define FLASH_DEV_INST		HL_DEV_GET(FLASH_DEV)
 
 #define EFUSE_DEV		DT_NODELABEL(efuse)
-#define EFUSE_DEV_INST		HL_DEV_GET(EFUSE_DEV);
+#define EFUSE_DEV_INST		HL_DEV_GET(EFUSE_DEV)
 
 #define EFUSE_SLV_SPI_DEV	DT_NODELABEL(efuse_slave_spi_gw)
-#define EFUSE_SLV_SPI_DEV_INST	HL_DEV_GET(EFUSE_SLV_SPI_DEV);
+#define EFUSE_SLV_SPI_DEV_INST	HL_DEV_GET(EFUSE_SLV_SPI_DEV)
 
 #define UART_DEV		DT_NODELABEL(uart0)
 #define UART_DEV_INST		HL_DEV_GET(UART_DEV)
